workflows/rnaseq/downstream/functional-enrichment.Rmd

```diff
@@ -29,10 +29,50 @@ id.convert <- TRUE
 # Convert gene IDs to those useable to clusterProfiler.
 # Docs: https://lcdb.github.io/lcdb-wf/rnaseq-rmd.html#bitrgenes
 # NOTE: Gene IDs are provided in what format?----------------------------------
-from.type <- "FLYBASE"
+from.type <- 'ENSEMBL'
+annotation_genus_species <- 'Mus musculus'
+annotation_key_override <- NA
+hub.cache <- '../../../include/AnnotationHubCache'
+orgdb <- get.orgdb(
+    annotation_genus_species,
+    cache=hub.cache,
+    annotation_key_override=annotation_key_override
+)
+# In place of AnnotationDbi::select, Use a wrapper function to run this command in order to catch the unmatching keys error
```

Comment:
I think this would fit better over in `lib/lcdbwf/R/helpers.R`. In fact, the `select.keyhander` could itself be created over there.